packages/react/src/patterns/blocks/ContentGroupSimple/README.md

```diff
@@ -0,0 +1,115 @@
+# Content Group - Simple
+
+> The Content Group - Simple pattern is to be utilized within IBM.com.
```

Comment:
Can we change this to something more descriptive, maybe something like
`The Content Group – Simple pattern includes a Heading and body copy, as well as optional media (image or video) and CTA components.`